libs/components/layout/testing/src/fluid-grid/row-harness.spec.ts

```diff
@@ -0,0 +1,59 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { SkyHelpTestingModule } from '@skyux/core/testing';
+
+import { FluidGridHarnessTestComponent } from './fixtures/fluid-grid-test.component';
+import { SkyRowHarness } from './row-harness';
+
+describe('Row test harness', () => {
+  async function setupTest(
+    options: {
+      dataSkyId?: string;
+    } = {},
+  ): Promise<{
+    rowHarness: SkyRowHarness;
+    fixture: ComponentFixture<FluidGridHarnessTestComponent>;
+    loader: HarnessLoader;
+  }> {
+    await TestBed.configureTestingModule({
+      imports: [FluidGridHarnessTestComponent, SkyHelpTestingModule],
+    }).compileComponents();
+
+    const fixture = TestBed.createComponent(FluidGridHarnessTestComponent);
+    const loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+
+    const rowHarness: SkyRowHarness = options.dataSkyId
+      ? await loader.getHarness(
+          SkyRowHarness.with({
+            dataSkyId: options.dataSkyId,
+          }),
+        )
+      : await loader.getHarness(SkyRowHarness);
+
+    return { rowHarness, fixture, loader };
```

Comment:
this loader is not being used, can we remove it from the return type?

Reply:
removed